frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/activity-card/cluster-expand-activity.tsx

```diff
@@ -0,0 +1,13 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import ActivityItem from '@console/shared/src/components/dashboard/activity-card/ActivityItem';
+import { K8sResourceKind } from '@console/internal/module/k8s';
+
+export const ClusterExpandActivity: React.FC = () => (
```

Comment:
We can reuse this too, but may be in future we want to modify these, hence keeping separate